### ISSUE TITLE
Rule Change: Add suggestions to `no-eq-null`

### ISSUE THREAD

**Issue by Pixel998** [eslint/eslint#21122]
### What rule do you want to change?

`no-eq-null`

### What change do you want to make?

Implement suggestions

### How do you think the change should be implemented?

Other

### Example code

```js
/* eslint no-eq-null: "error" */

if (value == null) {
  doSomething();
}

if (value != null) {
  doSomethingElse();
}
```

### What does the rule currently do for this code?

The rule reports both comparisons without providing suggestion to fix the code.

### What will the rule do after it's changed?

The rule will provide suggestions to replace the operator:

```js
if (value === null) {
  doSomething();
}

if (value !== null) {
  doSomethingElse();
}
```

### Participation

- [x] I am willing to submit a pull request to implement this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by DMartens** [comment]
I think this is a worthwile addition. Should the rule also provide a suggestion for an equal check using strict equality?:
```javascript
value == null // --> value === null || value === undefined
value != null // --> value !== null && value !== undefined
```

**Comment by mdjermanovic** [comment]
> I think this is a worthwile addition. Should the rule also provide a suggestion for an equal check using strict equality?:
> 
> ```js
> value == null // --> value == null || value == undefined
> value != null // --> value !== null && value !== undefined
> ```

In the first example, did you mean `value === null || value === undefined`? (`===` instead of `==`)

**Comment by DMartens** [comment]
Yes, I meant strict equals (edited post above accordingly).

**Comment by mdjermanovic** [comment]
Makes sense to me. @Pixel998 what do you think about https://github.com/eslint/eslint/issues/21122#issuecomment-5039269027?

**Comment by Pixel998** [comment]
Sounds good to me, marking as accepted. I think `eqeqeq` should also get this suggestion for `null` comparisons.
